backend/api/activity/index.js

```diff
@@ -43,5 +65,38 @@ activity.route('/create')
       status: statusCode,
     });
   });
+  
+/**
+ * Searches posts by criteria specified in post data
+ * TODO: Add search criteria
+ * @body {Number} numPosts - The number of posts to return
+ */
+activity.route('/search')
+  .get(async(req, res) => {
+	numPosts = 20;
+	if (Boolean(req.body.numPosts)) {
+		numPosts = req.body.numPosts;
+	}
+	let getobj = await Activity.find( {creationDateTime: {$exists: true}} ).sort({creationDateTime : -1}).limit(numPosts);
```

Comment:
As I mentioned in another comment, we need to use a `try...catch` block when we aren't using callbacks. This applies every time we are using mongoose to interface with the database.